Fix dem velocity verlet half steps (#2082)
chaos-polymtl/lethe: doc/source/parameters/dem/model_parameters.rst

All contact force models are described in the :doc:`../../theory/multiphase/dem/dem` section of the theory guide.

> * ``integration`` controls the integration method  used. Lethe supports ``euler`` (1st order) and ``velocity-verlet`` (2nd order) time-integrators. The velocity-verlet should be used at all times. The velocity Verlet scheme is staggered: it stores the velocities half a time step ahead of the positions. It therefore carries out an opening half velocity step at the first time step and a closing half velocity step, which synchronizes the velocities with the positions, at the end of the simulation (in pure DEM) or at the end of the DEM sub-iterations of every CFD time step (in CFD-DEM). Completing :math:`n` time steps consequently requires :math:`n+1` integration operations. Since the velocities are staggered in between, the velocities written to the visualization files during the simulation lag the positions by half a time step.

Reviewer: Velocity-verlet is written two different ways here. 

```suggestion
* ``integration`` controls the integration method  used. Lethe supports ``euler`` (1st order) and ``velocity-verlet`` (2nd order) time-integrators . ``velocity-verlet`` is the recommended integrator for most applications. The velocity Verlet scheme is staggered: it stores the velocities half a time step ahead of the positions. It therefore carries out an opening half velocity step at the first time step and a closing half velocity step, which synchronizes the velocities with the positions, at the end of the simulation (in pure DEM) or at the end of the DEM sub-iterations of every CFD time step (in CFD-DEM). Completing :math:`n` time steps consequently requires :math:`n+1` integration operations. Since the velocities are staggered in between, the velocities written to the visualization files during the simulation lag the positions by half a time step.
```
